spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/TestSparkV2Filters.java

```diff
@@ -39,6 +39,7 @@
 
 public class TestSparkV2Filters {
 
+  @SuppressWarnings("checkstyle:MethodLength")
```

Comment:
We should probably break this test up by operator :)

Reply:
I'd be okay doing that in a follow-up change as well.

Reply:
Split the internal test block to function `testV2Filter`.